cli/dcoscli/auth/main.py

```diff
@@ -178,9 +178,6 @@ def _trigger_client_method(
         auth.oidc_implicit_flow_auth(dcos_url)
     elif client_method == "dcos-usercredential-post-receive-authtoken" or \
             client_method == "dcos-credential-post-receive-authtoken":
-        if not username or not password:
```

Comment:
Can we make this:
```
if not (username and password):
...
```
It matches the text below more closely and is a bit easier to follow.